res/main/i18n/default/lists.json

```diff
@@ -5,8 +5,8 @@
 			"dc": "DC"
 		},
 		"mode": {
-			"goniometer": "Goniometer",
-			"triggered": "Triggered",
+			"goniometer": "Stereo image",
```

Comment:
It is a bad idea to make text associated with the key to differ the initial key sense.
If you want to change the label, please consider adding new key "stereo_image".